Dockerfile

```
@@ -59,10 +59,12 @@ FROM alpine:latest
 # - unzip: Required for PocketBase's zip features
 # - ca-certificates: Required for HTTPS connections
 # - wget: Required for health check
+# - su-exec: For privilege de-escalation in entrypoint
 RUN apk add --no-cache \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)